bridge/src/lib.rs

```diff
@@ -127,13 +83,30 @@ methods!(
         NilClass::new()
     }
 
-    fn create_worker(connection: AnyObject, namespace: RString, task_queue: RString) -> AnyObject {
-        let callback_sender = start_callback_loop();
+    fn init_runtime() -> AnyObject {
+        let runtime = Runtime::new(RUNTIME_THREAD_COUNT);
+
+        Module::from_existing("Temporal")
+            .get_nested_module("Bridge")
+            .get_nested_class("Runtime")
+            .wrap_data(runtime, &*RUNTIME_WRAPPER)
+    }
+
+    fn run_callback_loop() -> NilClass {
+        let runtime = rtself.get_data_mut(&*RUNTIME_WRAPPER);
+        runtime.run_callback_loop();
+
+        NilClass::new()
+    }
+
+    fn create_worker(runtime: AnyObject, connection: AnyObject, namespace: RString, task_queue: RString) -> AnyObject {
```

Comment:
I expect in the future this will be a config w/ lots of other options

Reply:
Yes, exactly!